.gitignore

```diff
@@ -52,4 +52,5 @@ typings/
 !.opensource
 
 # Build files
-dist
\ No newline at end of file
+dist
+package-lock.json
```

Comment:
This is just to ensure that if another contributor uses NPM instead of Yarn by mistake that their `package-lock.json` file isn't added to version control.